app/test/helpers/repositories.ts

```diff
@@ -120,6 +121,52 @@ export async function setupConflictedRepo(): Promise<Repository> {
   return repo
 }
 
+/**
+ * Setup a repository and create a merge conflict
+ *
+ * @returns the new local repository
+ *
+ * The current branch will be 'other-branch' and the merged branch will be
+ * 'master' in your test harness.
+ *
+ * The conflicted file will be 'foo'. There will also be uncommitted changes unrelated to the merge in 'perlin'.
+ */
+export async function setupConflictedRepoWithDirt(): Promise<Repository> {
```

Comment:
More descriptive name
```suggestion
export async function setupConflictedRepoWithUnrelatedUncommittedChange(): Promise<Repository> {
```

Reply:
changed in 6b8647a